operatorcourier/errors.py

```diff
@@ -0,0 +1,46 @@
+"""
+operatorcourier.errors
+
+Defines custom operator-courier exceptions.
+"""
+
+
+class OpCourierError(ValueError):
+    """Base class for operator-courier exceptions,
+    subclassed from ValueError for backward compatibility
+    """
+    pass
+
+
+class OpCourierBadYaml(OpCourierError):
+    """Invalid yaml file"""
+    pass
+
+
+class OpCourierBadArtifact(OpCourierError):
+    """File is a valid yaml, but not a valid CSV, CRD or Package"""
+    pass
+
+
+class OpCourierBadBundle(OpCourierError):
+    """Invalid bundle (e.g. missing CSV/CRD/Package).
+    Contains the info collected during validation of the bundle.
+    """
+    def __init__(self, msg, validation_dict):
```

Comment:
I'd call this `validation_results`, we shouldn't name variables with types in python